downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -84,6 +84,126 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 - Introduced GraphQL server-side caching support
 
+#### Introduction of the `data.jsondata` package
+
+- The [`data.jsondata`](https://lib.ballerina.io/ballerina/data.jsondata/latest/) package has been introduced to provide a set of APIs for JSON data conversions, data projection, and JSON navigation.
+
+    ##### Data Projection
```

Comment:
I wouldn't make these subsections. Maybe bullet points. We don't need the bullet point in L89.

Reply:
We should have bullet point in L89 since other modules also follow the same pattern